romanisim/image.py

```diff
@@ -227,23 +225,27 @@ def add_objects_to_image(image, objlist, xpos, ypos, psf,
     Parameters
     ----------
     image : galsim.Image
-        Image to which to add sources with associated WCS.
+        Image to which to add sources with associated WCS. Updated in place.
     objlist : list[CatalogObject]
         Objects to add to image
     xpos, ypos : array_like
         x & y positions of sources (pixel) at which sources should be added
     psf : galsim.Profile
         PSF for image
-    flux_to_counts_factor : float
+    flux_to_counts_factor : float or list
         physical fluxes in objlist (whether in profile SEDs or flux arrays)
         should be multiplied by this factor to convert to total counts in the
         image
+    exptimes: array_like
+        Exposure times to scale back to rate units
     bandpass : galsim.Bandpass
         bandpass in which image is being rendered.  This is used only in cases
         where chromatic profiles & PSFs are being used.
     filter_name : str
         filter to use to select appropriate flux from objlist.  This is only
         used when achromatic PSFs and sources are being rendered.
+    wcs : galsim.GSFitsWCS
+        WCS corresponding to image
```

Comment:
You can now remove the wcs argument, and always use image.wcs?

Reply:
I have removed it, though since l3_mos can be a datamodel, I had to tweak the code a little.

Reply:
I have removed this from image and not from l3